docs/source/agent-framework/core-service-agents/ieee-2030_5-agent/ieee-2030_5-agent.rst

```diff
@@ -1,13 +1,123 @@
+
+The following is an example of the agent configuration file:
+
+.. code-block:: yaml
+   # These are required in order for the agent to connect to the server.
+   cacertfile: ~/tls/certs/ca.crt
+   keyfile: ~/tls/private/dev1.pem
+   certfile: ~/tls/certs/dev1.crt
+   server_hostname: 127.0.0.1
+
+   # the pin number is used to verify the server is the correct server
+   pin: 111115
+
+   # Log the request and responses from the server.
+   log_req_resp: true
+
+   # SSL defaults to 443
+   server_ssl_port: 8443
+   # http port defaults to none
+   #server_http_port: 8080
+   # Number of seconds to poll for new default der settings.
+   default_der_control_poll: 60
+
+   MirrorUsagePointList:
+   # MirrorMeterReading based on Table E.2 IEEE Std 2030.5-18
+   - device_point: INV_REAL_PWR
+      mRID: 5509D69F8B3535950000000000009182
+      description: DER Inverter Real Power
+      roleFlags: 49
+      serviceCategoryKind: 0
+      status: 0
+      MirrorMeterReading:
+         mRID: 5509D69F8B3535950000000000009183
+         description: Real Power(W) Set
+         ReadingType:
+         accumulationBehavior: 12
+         commodity: 1
+         dataQualifier: 2
+         intervalLength: 300
+         powerOfTenMultiplier: 0
+         uom: 38
+   - device_point: INV_REAC_PWR
+      mRID: 5509D69F8B3535950000000000009184
+      description: DER Inverter Reactive Power
+      roleFlags: 49
+      serviceCategoryKind: 0
+      status: 0
+      MirrorMeterReading:
+         mRID: 5509D69F8B3535950000000000009185
+         description: Reactive Power(VAr) Set
+         ReadingType:
+         accumulationBehavior: 12
+         commodity: 1
+         dataQualifier: 2
+         intervalLength: 300
+         powerOfTenMultiplier: 0
+         uom: 38
+
+   # publishes on the following subscriptions will
+   # be available to create and POST readings to the
+   # 2030.5 server.
+   device_topic: devices/inverter1
+
+   # Nameplate ratings for this der client will be put to the
+   # server during startup of the system.
+   DERCapability:
+   # modesSupported is a HexBinary31 representation of DERControlType
+   # See Figure B.34 DER info types for information
+   # conversion in python is as follows
+   #   "{0:08b}".format(int("500040", 16))
+   #   '10100000000000001000000'  # This is a bitmask
+   # to generate HexBinary
+   #   hex(int('10100000000000001000000', 2))
+   #   0x500040
+   modesSupported: 500040
+   rtgMaxW:
+      multiplier: 0
+      value: 0
+   type: 0
+
+   DERSettings:
+   modesEnabled: 100000
+   setGradW: 0
+   setMaxW:
+      multiplier: 0
+      value: 0
+
+   # Note this file MUST be in the config store or this agent will not run properly.
+   point_map: config:///inverter_sample.csv
+
+
+.. note::
+   The ``point_map`` is configured through the config store at the location inverter_sample.csv.
+
+The following is an example of the mapping file (inverter_sample.csv):
+
+
```

Comment:
Document - assumptions, limitations. How this is different from previous 2030.5 agent. Mention that this was not tested with a actual 2030.5 device and hence should be used as template/starting point